R/with.r

```diff
@@ -128,9 +143,13 @@ set_libpaths <- function(paths) {
   invisible(old)
 }
 
+reset_libpaths <- function(paths) {
```

Comment:
@jimhester Thanks for letting me know. New development should continue there (https://github.com/jimhester/robustr/issues/1, https://github.com/jimhester/robustr/issues/2).